perceval/backends/core/hyperkitty.py

```diff
@@ -100,6 +115,14 @@ def has_caching(cls):
         """
         return False
 
+    @classmethod
```

Comment:
I think this backend doesn't need support for archiving at this moment. Let's see how it works first with the HTTP backends and then decided how to support this feature in Mailing Lists.